portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro

```diff
@@ -1,5 +1,51 @@
 definition {
 
+	macro assertCompanyLogIsolated {
```

Comment:
This should be in CompanyLogging.macro since it is more associated with Comapny Logging and not the JSON API (Since this macro will only be used in tests for CompanyLogging.test cases if that makes sense)